include/seqan3/core/detail/strong_type.hpp

```diff
@@ -432,9 +434,48 @@ class strong_type
         return get();
     }
     //!\}
+
+    /*!\name Formatted output.
+     * \brief Only available if the corresponding skill from seqan3::detail::strong_type_skill is added.
+     * \{
+     */
+
 private:
     //!\brief The underlying value, which is wrapped as a strong type.
     value_t value;
 };
 
+/*!\name Formatted output
+ * \relates seqan3::detail::strong_type
+ * \{
+ */
+
+/*!\brief Formatted output to a stream.
+ * \tparam stream_t The output stream to write to; must model seqan3::output_stream_over for the underlying type.
+ * \tparam underlying_type_t The underlying strong typed value type.
+ * \tparam derived_t The derived type of the crtp base class.
+ * \tparam skills The skills enabled with this strong type; must have seqan3::detail::strong_type_skill::stream
+ *                selected.
+ *
+ * \param[in,out] stream The output stream.
+ * \param[in] value The strong typed value to print.
+ *
+ * \details
+ *
+ * Prints the stored value of the given strong type.
+ *
+ * \returns `stream_t &` A reference to the given stream.
+ */
+template <typename stream_t, typename underlying_type_t, typename derived_t, strong_type_skill skills>
+//!\cond
+    requires output_stream_over<stream_t, underlying_type_t const &> &&
+             ((skills & strong_type_skill::stream) != strong_type_skill::none)
+//!\endcond
+stream_t & operator<<(stream_t & stream, strong_type<underlying_type_t, derived_t, skills> const & value)
```

Comment:
Why did you not overload `debugstream`?

I'm not a huge fan of overloading every possible stream operator.

I would prefer a specific overload for `debugstream`.

Reply:
I found it weird if we say the skill allows streaming the type but then only restrict it to the debug stream. I am also in slight preference for making the strong type a public type at some point.

Reply:
The question is why strong types are not always printable if the underlying type is printable?

Please just overload debug_stream :pleading_face:

Reply:
That is exactly the design of the strong types. They are not supposed to automatically inherit the behavior of the underlying type but it needs to be selected individually. If you want the type to inherit all its features just use a type alias. 
It would be ok, I guess as, long as the strong type is detail to make it only debug streambale. It would not be ok if it is public. Something we should consider for the future if we are beginning to use strong types in more public interfaces. Since then the user knows what can be done with the interface. I will make it now explicit, but that should become a generic stream overload if it goes public.